### ISSUE TITLE
Add "Hello World" in Somali to app

### ISSUE THREAD

**Issue by tacostreets** [Titane73/it115-a5-g2-app#37]
Please look at the app once it is merged and make a branch to add a Somali version of "Hello World". Add it to the drop down list and the display. This is lower priority. 

Please add me as reviewer and add this issue number to the end of your commit description when you commit your branch.
